src/wp-includes/https-detection.php

```diff
@@ -109,12 +109,22 @@ function wp_update_https_detection_errors() {
 
 	$support_errors = new WP_Error();
 
+	$headers = array();
+	$headers['Cache-Control'] = 'no-cache';
+
+	/*
+	 * Include basic auth in loopback requests. Note that this will only pass along basic auth when user is
+	 * initiating the test. If a site requires basic auth, the test will fail when it runs in WP Cron as part of
+	 * wp_site_health_scheduled_check. This logic is copied from WP_Site_Health::can_perform_loopback().
```

Comment:
```suggestion
	 * Include basic auth in loopback requests. Note that this will only pass along basic auth when user is
	 * initiating the test. If a site requires basic auth, the test will fail when it runs in WP Cron as part of
	 * wp_site_health_scheduled_check.
```

No need to include the source in the committed code, it's helpful for the review only.